verification/article/Makefile

```
@@ -0,0 +1,2 @@
+@generate-pdf:
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)